website/content/en/preview/concepts/setup.md

```diff
@@ -0,0 +1,551 @@
+ROLES   arn:aws:iam::111111111111:role/MyRoleForInstances      2020-12-10T19:45:30+00:00       /       AROA6FGHHBQSQ5NKPJT2H   MyRoleForInstances
+ASSUMEROLEPOLICYDOCUMENT        2012-10-17
+STATEMENT       sts:AssumeRole  Allow
+PRINCIPAL       ec2.amazonaws.com
+```
+
+## KarpenterNodeRole
+
+This section creates the node role that is attached to the `KarpenterNodeInstanceProfile` instance profile created earlier.
+Given a cluster name of `joe-karpenter-demo`, this role would end up being named `"KarpenterNodeRole-joe-karpenter-demo`.
+
+```
+  KarpenterNodeRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: !Sub "KarpenterNodeRole-${ClusterName}"
+      Path: /
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                !Sub "ec2.${AWS::URLSuffix}"
+            Action:
+              - "sts:AssumeRole"
+      ManagedPolicyArns:
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEKS_CNI_Policy"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEKSWorkerNodePolicy"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonSSMManagedInstanceCore"
+```
+
+The role created here includes several AWS managed policies, which are designed to provide permissions for specific uses needed by the nodes to work with EC2 and other AWS resources. These include:
+
+* [AmazonEKS_CNI_Policy](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonEKS_CNI_Policy.html): Provides the permissions that the Amazon VPC CNI Plugin needs to configure EKS worker nodes.
+* [AmazonEKSWorkerNodePolicy](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonEKSWorkerNodePolicy.html): Lets Amazon EKS worker nodes connect to EKS Clusters.
+* [AmazonEC2ContainerRegistryReadOnly](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonEC2ContainerRegistryReadOnly.html): Allows read-only access to repositories in the Amazon EC2 Container Registry.
+* [AmazonSSMManagedInstanceCore](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonSSMManagedInstanceCore.html): Adds AWS Systems Manager service core functions for Amazon EC2.
+
+# Karpenter Controller Authorization in cloudformation.yaml
+
+This section sets the permissions that the Karpenter Controller has to create and manage EC2 and other AWS resources.
+In particular, the section creates a service account (karpenter) that is combined with the KarpenterControllerPolicy.
+The permissions here go beyond the permissions assigned to Karpenter nodes in the previous section.
+
+The resources defined in this section include:
+
+* KarpenterControllerPolicy
+* KarpenterInterruptionQueue
```

Comment:
This and the other things down to L142 don't apply to the controller authorization

Reply:
Thanks. I forgot to fix that. Done.